app/src/androidTest/java/com/example/henzoshimada/feeltrip/FeelTripTests.java

```diff
@@ -174,11 +213,11 @@ public void testGetDate() {
 
 /*
     public void testSetImageFile() {
-        //TODO: Figure out what data type an image file is stored in
```

Comment:
We're taking the image, converting it to String, then pushing to elasticsearch. So the only thing the controller has to handle is the String, everything else we'll handle before we push/after we pull